JustSaying/Messaging/MessagePublisherExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using JustSaying.Models;
+
+namespace JustSaying.Messaging
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class MessagePublisherExtensions
+    {
+        public static Task PublishAsync(this IMessagePublisher publisher, Message message)
+        {
+            return publisher.PublishAsync(message, CancellationToken.None);
+        }
+
+        public static async Task PublishAsync(this IMessagePublisher publisher,
```

Comment:
One for `(Message, CancellationToken)` too?

Reply:
It's there, line 17-27

Reply:
It was at the bottom of the PR when I was reviewing it, hence I didn't see as I was commenting.